### ISSUE TITLE
Thanks so much Revival Team!

### ISSUE THREAD

**Issue by WiseDino** [FossilsArcheologyRevival/FossilArcheology1.7#109]
Hi everyone first post and i would like to thank the team for the pteranodon flying it looks amazing. Some suggestions i have are don't let babies fly and would love to see a feature where they actually dive or land instead of just floating down. You're probably already working on that but this mod is the main reason i play mine craft anymore so keep up the GREAT work. Thanks.

**Comment by Adsper** [comment]
I thought babies were already grounded?

**Comment by tkdberger** [comment]
Please keep these topics to the forum post , which can be found [here.](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2398119-fossils-and-archeology-revival-build-7-1-the) Please close this issue, or hopefully the mods will close it for you, as this "issue" is bumping down actual issues.
Tl;Dr: This is an issue tracker, not a forum for casual discussion.

**Comment by WiseDino** [comment]
If you'll notice the wiki page that says for any issues AND suggestions go to our GitHub page. Thank you

**Comment by 4f6f3b** [comment]
Hey tkdberger, thanks for trying to keep things neat, but yeah we also use this space for suggestions as well as long as they're on topic. Much easier to keep track of than the forum thread where it tends to derail and things get buried.

**Comment by tkdberger** [comment]
Ah, forgive me then. I swear I have minor OCD sometimes...

On 5/3/15, 4f6f3b <notifications@github.com> wrote:
> Hey tkdberger, thanks for trying to keep things neat, but yeah we also use
> this space for suggestions as well as long as they're on topic. Much easier
> to keep track of than the forum thread where it tends to derail and things
> get buried.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/FossilsArcheologyRevival/FossilArcheology1.7/issues/109#issuecomment-98554427


-- 
 Sincerely,
Caleb

**Comment by Alex-the-666** [comment]
Babies are already grounded and cannot fly.
